Application/EdFi.Ods.Api/Helpers/TypeHelper.cs

```diff
@@ -41,7 +41,7 @@ private static List<(Assembly assembly, Type type)> GetImplementationsOf<T>()
         {
             var allModules = AppDomain.CurrentDomain.GetAssemblies()
                 .SelectMany(
-                    a => a.GetTypes()
+                    a => a.GetExportedTypes()
```

Comment:
workaround for [compatibility issue](https://github.com/dotnet/SqlClient/issues/2228)